R/with.r

```diff
@@ -128,9 +143,13 @@ set_libpaths <- function(paths) {
   invisible(old)
 }
 
+reset_libpaths <- function(paths) {
```

Comment:
There is an inconsistency: with_path will *append* the new path to `PATH` if `add` is set, but with_lib should *prepend*. This inconsistency will be difficult to communicate. Options:

- Prepend to `PATH` if `add` is set
- Use parameter `action = c("replace", "prefix", "suffix")` instead of `add` for with_path and with_lib

Please advise.

While we're at it: The `with` functions are tremendously useful on their own, how about extracting them to a separate package (with less dependencies ;-) )?